examples/CMakeLists.txt

```diff
@@ -108,3 +108,36 @@ else()
             "UMF_BUILD_LEVEL_ZERO_PROVIDER, UMF_BUILD_LIBUMF_POOL_DISJOINT and "
             "UMF_ENABLE_POOL_TRACKING to be turned ON - skipping")
 endif()
+
+if(LINUX)
+    set(BASE_NAME ipc_shm_osmemprov)
```

Comment:
could we name it ipc_os_prov?

Reply:
Maybe `ipc_shm_os_prov` ?

Reply:
IPC always uses shared memory, so I think it is redundant